app/components/animated_number/index.tsx

```diff
@@ -76,7 +100,7 @@ const AnimatedNumber = ({
                     )}
                     {Array.from(animateToNumberString, Number).map((_, index) => (
                         <View
-                            key={`${index.toString()}`}
+                            key={`${animateToNumberString.length - index}`}
```

Comment:
Was using index as a key, but changing from 99 => 199, were causing the original two views to be re-rendered when it is not necessary.

By using the reverse index as key, hopefully the first 2 views will stay intact, and React only have to add 1 additional View tree.

Keep in mind, there are may Views (10) within each View, so not causing the parent View to re-render is a good thing.